worker/src/durable.rs

```diff
@@ -737,12 +768,40 @@ impl DurableObject for Chatroom {
 }
 ```
 */
+
 #[async_trait(?Send)]
 pub trait DurableObject {
     fn new(state: State, env: Env) -> Self;
+
     async fn fetch(&mut self, req: Request) -> Result<Response>;
+
     #[allow(clippy::diverging_sub_expression)]
     async fn alarm(&mut self) -> Result<Response> {
         unimplemented!("alarm() handler not implemented")
     }
+
+    #[allow(unused_variables, clippy::diverging_sub_expression)]
+    async fn websocket_message(
+        &mut self,
+        ws: WebSocket,
+        message: WebSocketIncomingMessage,
+    ) -> Result<()> {
+        unimplemented!("websocket_message() handler not implemented")
```

Comment:
So if I'm writing a DO in rust and I use websocket hibernation, I have to implement these otherwise when they're dispatched we panic? If you're aiming to match `Workerd`s behavior we [silently drop the event](https://github.com/cloudflare/workerd/blob/fa0e7bf74b406a241888c87f8327af917a6d7d0d/src/workerd/api/global-scope.c%2B%2B#L526), but I see the value in warning.

Reply:
I tend to agree with you, however I'm just matching the existing implementation of `alarm` above.

```rs
#[allow(clippy::diverging_sub_expression)]
    async fn alarm(&mut self) -> Result<Response> {
        unimplemented!("alarm() handler not implemented")
    }
```

If you'd like this approach changed, I think we should also change the `alarm` implementation to match.